patroni/dcs/__init__.py

```diff
@@ -1336,15 +1334,15 @@ class AbstractDCS(abc.ABC):
     _SYNC = 'sync'
     _FAILSAFE = 'failsafe'
 
-    def __init__(self, config: Dict[str, Any]) -> None:
+    def __init__(self, config: Dict[str, Any], mpp: 'AbstractMPP') -> None:
```

Comment:
Do we always have to pass a `AbstractMPP` instance?

Reply:
Yes, we have to.
It needs to get an idea whether we work with just Postgres/Patroni cluster or with MPP (Citus, or similar).

Reply:
Right, I forgot to remove this comment, sorry. I noticed the `Null` stuff later.
Thanks for clarifying IAC.